oap-server/server-starter/src/main/resources/ui-initialized-templates/aws_eks/aws_eks-cluster.json

```diff
@@ -443,38 +451,35 @@
             "dashboardName": "AWS_EKS-node",
             "fontSize": 12
           },
-          "metrics": [
-            "eks_cluster_node_cpu_utilization",
-            "eks_cluster_node_memory_utilization",
-            "eks_cluster_node_network_tx_errors",
-            "eks_cluster_node_network_rx_errors"
+          "expressions": [
+            "avg(eks_cluster_node_cpu_utilization/100)",
+            "avg(eks_cluster_node_memory_utilization/100)",
```

Comment:
Please confirm if cpu and memory usage needs div 100, the previous calculation is only average